cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java

```diff
@@ -231,6 +234,10 @@ public void stop() {
     }
   }
 
+  protected void setLogLevel(Map<String, String> args, LogAppenderInitializer logAppenderInitializer) {
+    SystemArguments.setLogLevel(args, logAppenderInitializer);
```

Comment:
Get rid of this kind of one line helper. Sub-classes don't need to override this. They can simply set the log level again to override.